flipp/src/app/components/AppContainer.js

```diff
@@ -1,7 +1,31 @@
+"use client";
 import CardContainer from "./CardContainer";
 import Header from "./Header";
 import ButtonContainer from "./ButtonContainer";
+import { useState } from "react";
 export default function AppContainer() {
+  const [flip, setFlip] = useState(false);
+  const [currentCard, setCurrentCard] = useState(0);
```

Comment:
And also have to keep track of the current card that we are showing when previous and next buttons are used (which onClick event handlers change the state of the currentCard variable)